package.json

```diff
@@ -47,5 +48,8 @@
     "browserslist": [
         "defaults",
         "IE 8"
-    ]
+    ],
+    "dependencies": {
+        "http-server": "^14.1.1"
+    }
```

Comment:
Suggestion: rather than add dependencies to the main package, it might be nice to create a separate `package.json` file inside `/demo` that can be responsible for demo-only dependencies. See the traffic cop repo as an example.